build/docker/build-image.sh

```diff
@@ -18,9 +18,12 @@ fi
 TAG=${TAG:-devel}
 
 BUILD_ARGS=
+if [ -n ${EXTRA_BUILD_ARGS} ]; then
```

Comment:
would it make sense to add optional parameter to the script instead of relying on environment(global?) variables?

Reply:
v2 does this.